feature-detects/battery.js

```diff
@@ -16,5 +16,5 @@
 Detect support for the Battery API, for accessing information about the system's battery charge level.
 */
 define(['Modernizr', 'prefixed'], function(Modernizr, prefixed) {
-  Modernizr.addTest('batteryapi', !!prefixed('battery', navigator), {aliases: ['battery-api']});
+  Modernizr.addTest('batteryapi', !!prefixed('getBattery', navigator), {aliases: ['battery-api']});
```

Comment:
as both shipped in browsers, we want to test for both.

Reply:
@patrickkettner Ok so a new file should be created?